doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -432,24 +368,6 @@ Requests to the environment
 
          SearchRewrite (_ + _ + _).
 
-   .. cmdv:: SearchRewrite @term inside {+ @qualid }
-
-      This restricts the search to constructions defined in the modules
-      named by the given :n:`qualid` sequence.
-
-   .. cmdv:: SearchRewrite @term outside {+ @qualid }
-
-      This restricts the search to constructions not defined in the modules
-      named by the given :n:`qualid` sequence.
-
-   .. cmdv:: @selector: SearchRewrite @term
-
-      This specifies the goal on which to
-      search hypothesis (see Section :ref:`invocation-of-tactics`).
-      By default the 1st goal is
-      searched. This variant can be combined with other variants presented
-      here.
-
 .. note::
```

Comment:
It doesn't really make sense that this `table` must be presented in a note.

Reply:
Got it.

BTW, if you can select multiple lines it's a bit easier for me to find what you're referring to. `.. note::` isn't a great search term and the line numbers get thrown off as I make changes going from top to bottom (was 357 for me vs 371 shown above).

![image](https://user-images.githubusercontent.com/1253341/78463405-0a197f00-7691-11ea-83ef-02d69c7236fa.png)